charts/nri-metadata-injection/tests/nodeselector_test.yaml

```diff
@@ -0,0 +1,23 @@
+suite: test nodeSelector setup
+templates:
+  - templates/deployment.yaml
+release:
+  name: release
+  namespace: ns
+tests:
+  - it: nodeSelector is not set by default
+    set:
+      cluster: my-cluster
+      asserts:
+        - isNull:
+            path: spec.template.spec.nodeSelector
+  - it: NodeSelector custom value
+    set:
+      cluster: my-cluster
+      nodeSelector:
+        disktype ssd
+      asserts:
+        - equal:
+            path: spec.template.spec.nodeSelector
+            value:
+              distktype: ssd
```

Comment:
Just out of curiosity... Why is this not failing?

It should be `disktype: ssd` in the line 18...

Could you dig it a bit?

Reply:
Because it was all wrong! The `assert` field indentation was wrong, it was inside the previous `set` block, so there was no assertions defined 😓 

Anyway, I am removing the file to avoid _re-testing_ as suggested.

Thanks!

Reply:
Oh, you are right! I didn't realize. Awesome.